docs/guides/latest-updates.mdx

```diff
@@ -29,6 +29,23 @@ Jump to [Qiskit SDK](#latest-qiskit-sdk) | [Qiskit Runtime client](#latest-qiski
 
 This section summarizes the recent enhancements and new features for IBM Quantum Platform.
 
+**24 April 2025**
+
+- Instance limits: Individuals can view [available time and usage](https://quantum.cloud.ibm.com/instances), which displays alerts when approaching account or instance limits.
```

Comment:
```suggestion
- Instance limits: Individuals can view [available time and usage,](https://quantum.cloud.ibm.com/instances) which displays alerts when approaching account or instance limits.
```

Reply:
What is the "account" limits?  I know there are allocation limits, cost limits, and instance limits, but I don't see where we talk about account limits.

Reply:
@lerongil @JulesMurphy is 'account limits' the right term?

Reply:
This can be replaced with" 

Individuals can view [available and remaining usage,](https://quantum.cloud.ibm.com/instances) which displays alerts when approaching remaining usage.

Reply:
"approaching remaining usage" seems vague, since philosophically we are in a constant state of approaching our remaining usage.... 🤣  how about "when approaching usage limits" or just "when approaching limits"?